cmake/Windows/NSIS.template.in

```diff
@@ -291,7 +284,7 @@ Section "TOPP tools" SEC_TOPP
     #open uninstall log file
     !insertmacro UNINSTALL.LOG_OPEN_INSTALL
 
-    # Since Qt 5.14 we probably do not need qt.conf if the plugins are in the standard location "plugins"
+    # Since Qt 5.14 we do not need qt.conf if the plugins are in the standard location "plugins"
     # Qt is built "-relocatable" since then.
     #File "${OPENMSBINDIR}\qt.conf"

```

Comment:
No surprise that older Qts don't find the plugins when this is commented out. But if Qt 5.15 is available everywhere we build installers I think I'm okay with requiring it.
But probably is not too hard to fix, either.

Reply:
jep. But if we can avoid the qt.conf, I'd avoid it.